### PR TITLE
Cleanup IDE issues for AddContentApi

### PR DESCRIPTION
**Pull request by lukstbit**
## Purpose / Description

Cleanups for the `AddContentApi` class to remove kotlin cleanups and IDE reported issues. There are several TODOs remaining but those will be fixed in a separate PR as they are related to the type system(possible impact on the public API).

### CODE REVIEW

File: api/src/main/java/com/ichi2/anki/api/AddContentApi.kt [hunk 1]
```diff
@@ -602,7 +545,6 @@ class AddContentApi(context: Context) {
                         FlashCardsContract.AUTHORITY,
                         PackageManager.GET_META_DATA
                     )
```
Comment (david-allison) [inline api/src/main/java/com/ichi2/anki/api/AddContentApi.kt:547]: nit: unnecessary

File: api/src/main/java/com/ichi2/anki/api/AddContentApi.kt [hunk 2]
```diff
@@ -416,13 +378,10 @@ class AddContentApi(context: Context) {
         get() {
             // Get the current model
             val uri = Uri.withAppendedPath(Model.CONTENT_URI, Model.CURRENT_MODEL_ID)
-            val singleModelCursor = mResolver.query(uri, null, null, null, null) ?: return -1L
-            val modelId: Long
-            modelId = try {
+            val singleModelQuery = mResolver.query(uri, null, null, null, null) ?: return -1L
+            val modelId: Long = singleModelQuery.use { singleModelCursor ->
```
Comment (david-allison) [inline api/src/main/java/com/ichi2/anki/api/AddContentApi.kt:382]: can this be flipped to a `return`?

**Review by david-allison — APPROVED** [review]
Looks great! Only nitpicks

File: api/src/main/java/com/ichi2/anki/api/AddContentApi.kt (hunk 2 above)
Comment (lukstbit) [reply api/src/main/java/com/ichi2/anki/api/AddContentApi.kt]: Yes, I missed that.

**Review by BrayanDSO — APPROVED** [review]
much nicer, thanks

**Pull request merged**